libs/langchain-cloudflare/src/chat_models.ts

```diff
@@ -109,9 +193,14 @@ export class ChatCloudflareWorkersAI
   /**
    * Get the parameters used to invoke the model
    */
-  invocationParams(_options?: this["ParsedCallOptions"]) {
+  invocationParams(options?: this["ParsedCallOptions"]) {
+    const tools =
+      options?.tools && options.tools.length
```

Comment:
Passing `tools: undefined` is fine right?

Reply:
yes